doc/modules/manifold.rst

```diff
@@ -502,6 +502,7 @@ three parameters that control the optimization of t-SNE:
 * early exaggeration factor
 * learning rate
 * maximum number of iterations
+* angle
```

Comment:
Yep, agreed that high values of `angle` should negatively impact the quality of the embedding, and so is as relevant as the learning rate or number of iterations.